Dfe.Academies.External.Web/wwwroot/js/govuk-frontend-2033453473833bc9b69ee0edc2f0e481.js

```diff
@@ -0,0 +1,2 @@
e("aria-selected"),t.removeAttribute("tabindex");t=this.getPanel(t);t.removeAttribute("role"),t.removeAttribute("aria-labelledby"),t.classList.remove(this.jsHiddenClass)},p.prototype.onTabClick=function(t){if(!t.target.classList.contains("govuk-tabs__tab"))return!1;t.preventDefault();var e=t.target,t=this.getCurrentTab();this.hideTab(t),this.showTab(e),this.createHistoryEntry(e)},p.prototype.createHistoryEntry=function(t){var e=this.getPanel(t),n=e.id;e.id="",this.changingHash=!0,window.location.hash=this.getHref(t).slice(1),e.id=n},p.prototype.onTabKeydown=function(t){switch(t.keyCode){case this.keys.left:case this.keys.up:this.activatePreviousTab(),t.preventDefault();break;case this.keys.right:case this.keys.down:this.activateNextTab(),t.preventDefault()}},p.prototype.activateNextTab=function(){var t,e=this.getCurrentTab(),n=e.parentNode.nextElementSibling;(t=n?n.querySelector(".govuk-tabs__tab"):t)&&(this.hideTab(e),this.showTab(t),t.focus(),this.createHistoryEntry(t))},p.prototype.activatePreviousTab=function(){var t,e=this.getCurrentTab(),n=e.parentNode.previousElementSibling;(t=n?n.querySelector(".govuk-tabs__tab"):t)&&(this.hideTab(e),this.showTab(t),t.focus(),this.createHistoryEntry(t))},p.prototype.getPanel=function(t){return this.$module.querySelector(this.getHref(t))},p.prototype.showPanel=function(t){this.getPanel(t).classList.remove(this.jsHiddenClass)},p.prototype.hidePanel=function(t){this.getPanel(t).classList.add(this.jsHiddenClass)},p.prototype.unhighlightTab=function(t){t.setAttribute("aria-selected","false"),t.parentNode.classList.remove("govuk-tabs__list-item--selected"),t.setAttribute("tabindex","-1")},p.prototype.highlightTab=function(t){t.setAttribute("aria-selected","true"),t.parentNode.classList.add("govuk-tabs__list-item--selected"),t.setAttribute("tabindex","0")},p.prototype.getCurrentTab=function(){return this.$module.querySelector(".govuk-tabs__list-item--selected .govuk-tabs__tab")},p.prototype.getHref=function(t){t=t.getAttribute("href");return t.slice(t.indexOf("#"),t.length)},i((t="undefined"!=typeof(t=void 0!==t?t:{}).scope?t.scope:document).querySelectorAll('[data-module="govuk-button"]'),function(t){new o(t).init()}),i(t.querySelectorAll('[data-module="govuk-accordion"]'),function(t){new e(t).init()}),i(t.querySelectorAll('[data-module="govuk-details"]'),function(t){new s(t).init()}),i(t.querySelectorAll('[data-module="govuk-character-count"]'),function(t){new r(t).init()}),i(t.querySelectorAll('[data-module="govuk-checkboxes"]'),function(t){new a(t).init()}),new l(t.querySelector('[data-module="govuk-error-summary"]')).init(),new u(t.querySelector('[data-module="govuk-header"]')).init(),i(t.querySelectorAll('[data-module="govuk-notification-banner"]'),function(t){new c(t).init()}),i(t.querySelectorAll('[data-module="govuk-radios"]'),function(t){new d(t).init()}),new h(t.querySelector('[data-module="govuk-skip-link"]')).init(),i(t.querySelectorAll('[data-module="govuk-tabs"]'),function(t){new p(t).init()})}();
```

Comment:
I assume we want this file to be deleted by Moses' PR?

Reply:
I hope so, yes, I suspect it's part of the standard gov UK NPM package